### PR TITLE
jail.local.j2: sort service.items to ensure idempotency

### PR DESCRIPTION
**Pull request by maxlareo**
The promised pull request made in the issue https://github.com/Oefenweb/ansible-fail2ban/issues/48

### CODE REVIEW

File: templates/etc/fail2ban/jail.local.j2 [hunk 1]
```diff
@@ -105,7 +105,7 @@ action = {{ fail2ban_action }}
 {% for service in fail2ban_services %}
 [{{ service.name }}]
 enabled = {{ service.enabled | default(true) | bool | to_json }}
-{% for option, value in service.items() %}
+{% for option, value in service.items()|sort %}
```
Comment (tersmitten) [inline templates/etc/fail2ban/jail.local.j2]: Shouldn't this be [dictsort](http://jinja.pocoo.org/docs/2.10/templates/#dictsort)?

**Comment by tersmitten** [comment]
Fixes #48

File: templates/etc/fail2ban/jail.local.j2 (hunk 1 above)
Comment (maxlareo) [reply templates/etc/fail2ban/jail.local.j2]: it seems that using dictsort give me the same result.
Using this : `{% for option, value in service|dictsort %}`

I am still new in the jinja2 templating world and I don't know which solution is the best suited here. But I think that, as you propose, dictsort seems to be made for this kind of situation, dealing with dict.

I will let you choose the right way here ;-)

Comment (tersmitten) [reply templates/etc/fail2ban/jail.local.j2]: I'll have a look at it

Comment (tersmitten) [reply templates/etc/fail2ban/jail.local.j2]: https://www.tutorialspoint.com/python/dictionary_items.htm

> `items()` returns a list of dict's (key, value) `tuple` pairs

So `sort` seems to be correct

**Pull request merged**